ctx.go

```diff
@@ -383,6 +383,42 @@ func (c *DefaultCtx) HasBody() bool {
 	return len(c.fasthttp.Request.Body()) > 0
 }
 
+// UpdateParam overwrites a route parameter value by name.
+// If the parameter name does not exist in the route, this method does nothing.
+func (c *DefaultCtx) UpdateParam(name, value string) {
+	// If no route is matched, there are no parameters to update
+	if c.route == nil {
+		return
+	}
+
+	// Normalize wildcard (*) and plus (+) tokens to their internal
+	// representations (*1, +1) used by the router.
+	if name == "*" || name == "+" {
+		name += "1"
+	}
+
+	for i, param := range c.route.Params {
+		// Prevent out-of-bounds access if route params exceed allocated values
+		if i >= maxParams {
+			return
+		}
+		if c.app.config.CaseSensitive {
+			if param == name {
+				c.values[i] = value
+				return
+			}
+		} else {
+			if len(param) != len(name) {
```

Comment:
This check can be removed, see https://github.com/gofiber/utils/blob/master/byteseq.go#L9